docs/gettingstarted/getting-started-runtime-configuration.md

```diff
@@ -51,19 +51,32 @@ net.ipv4.ip_forward=1
 net.ipv4.ip_forward=1
 ```
 ## Select a Container Runtime and SC4S Configuration
+If no additional operating system requirements are specified, launching the selected configuration should be independent of the operating system as long as the containerization engine is installed correctly.
 
-| Container Runtime and Orchestration                               | Operating Systems                                                                   |
+Working on CentOS 8 or RHEL 8 is particularly recommended.
+
+For Podman configuration, it has been tested on the following distributions:
```

Comment:
The Podman configuration is supported on the following distributions: Note - can we say supported instead of "tested" here? Or is the word "testing" very specific to this case?

Reply:
yes, thank you